app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
```

Comment:
- [ ] According to the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), associations between models are set up correctly, something like so: 

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```
Please kindly fix that.

Reply:
@t-yanick Thanks for the comment. Kindly I was wondering to ask you if it is possible to use the `foreign_key: 'user_id'` the foreign_key was set by `user_id`